src/main/kotlin/org/jitsi/utils/logging2/LoggerExtensions.kt

```diff
@@ -48,9 +48,11 @@ fun <T : Any> T.createChildLogger(
  * by stripping any companion object class identifier, if present.
  */
 fun <T : Any> getClassForLogging(javaClass: Class<T>): Class<*> {
-    return (javaClass.enclosingClass?.takeIf {
-        it.kotlin.companionObject?.java == javaClass
-    } ?: javaClass)
+    return (
+        javaClass.enclosingClass?.takeIf {
+            it.kotlin.companionObject?.java == javaClass
+        } ?: javaClass
+        )
```

Comment:
Ugh, I hate ktlint's formatting for this.  Does it really not want that final closing parenthesis on the same indent as the `return`?

Reply:
That's what `ktlint -F` produced, but I just simplified it.